modules/doc/content/getting_started/examples_and_tutorials/tutorial03_verification/step03_analytical_solution.md

```diff
@@ -0,0 +1,247 @@
+# Problem with Analytical Solution
+
+## Introduction
+
+The best method for verifying a simulation is comparison with a known mathematical solution.
+For the heat equation, as defined in [step02_fem_convergence.md], one such solution
+shall be considered to verify a model. The verification process shall be broken into four parts:
+
+1. Define a problem, with a known exact solution.
+2. Simulate the problem.
+3. Compute the error between the known solution and simulation.
+4. Perform convergence study.
+
+## Problem Statement
+
+[!cite](incropera1996fundamentals) presents a one-dimensional solution for the heat equation as:
+
+!equation id=tutorial03-analytical-solution
+T(t,x) = T_0 + \frac{\dot{q}}{k} \Bigg\lbrack 2 \sqrt{\frac{\alpha t}{\pi}}
+         \exp\bigg\lparen{\frac{-x^2}{4 \alpha t}}\bigg\rparen -
+         x \bigg\lparen 1-\textrm{erf}\bigg\lparen\frac{x}{2\sqrt{\alpha t}} \bigg\rparen \bigg\rparen \Bigg\rbrack,
+
+where $T$ is the temperature, $T_0$ is the initial temperature, $t$ is time, $x$ is position,
+$\dot{q}$ is the heat source, $k$ is thermal conductivity, and $\alpha$ is the thermal diffusivity.
+
+Thermal diffusivity is defined as the ratio of thermal conductivity ($k$) to the product of density ($\rho$)
+and specific heat capacity ($c$). The system is subjected to Neumann boundary conditions at the
+left boundary ($x=0$) and the natural boundary condition on
```

Comment:
natural = zero flux here? Natural could mean different things for FE FV

Reply:
added (zero flux)